react/components/SKUSelector/README.md

```diff
@@ -1,73 +1,71 @@
 
-To import it into your code: 
-```js
-import SKUSelector from 'vtex.store-components/SKUSelector'
-```
+`SKUSelector` is a VTEX Component that is resposible to handle events of sku selection for a product. This component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
+
+## Table of Contents
+- [Usage](#usage)
+  - [Blocks API](#blocks-api)
+    - [Configuration](#configuration)
+  - [Styles API](#styles-api)
+    - [CSS Namespaces](#css-namespaces)
 
 ## Usage
 
-You can use it into your code like a React component with the jsx tag: `<SKUSelector />`. 
-
-```jsx
-<SKUSelector
-  title={'SKUSelector'}
-  skuItems={[
-    {
-      name: 'SKU',
-      images: [
-        {
-          imageUrl: 'www.image.url.com',
-          imageLabel: 'This is an image',
-        }
-      ],
-      specs: [
-        name: 'Color',
-        categories: [
-          { name: 'Red' },
-          { name: 'Blue' }
-        ]
-      ]
-    }
-  ]}
-/>
+You should follow the usage instruction in the main [README](/README.md#usage).
+
+Then, add `sku-selector` block into your app theme, as we do in our [Product Details app](https://github.com/vtex-apps/product-details/blob/master/store/blocks.json). 
+
+### Blocks API
+
+When implementing this component as a block, various inner blocks may be available. The following interface lists the available blocks within `SKUSelector` and describes if they are required or optional.
+
+```json
+  "sku-selector": {
+    "component": "SKUSelector"
+  }
 ```
-### Props
 
-| Prop name          | Type       | Description                                                                 |
-| ------------------ | ---------- | --------------------------------------------------------------------------- |
-| `title`            | `String!`  | SKU Selector title                                                          |
-| `skuItems`         | `Array!`   | SKU list                                                                    |
-| `defaultIndex`     | `Number`   | Default SKU Selection in case of is not the first item                      |
-| `onSKUSelected`    | `Function` | Function that is called when a SKU is selected                              |
+For now this block does not have any required or optional blocks.
+
+#### Configuration
+
+Through the Storefront, you can change the `SKUSelector` behavior and interface. However, you also can make in your theme app, as Store theme does.
 
-- `skuItems` shape
+| Prop name | Type | Description | Default value |
+| --------- | ---- | ----------- | ------------- |
+| productSlug | String | Product's slug | Required | 
+| skuSelected | SKU | SKU selected | Required |
+| skuItems | Array(SKU) | List of SKU Items | Required |
+| onSKUSelected | Function | Callback that is called when a SKU is selected | Function that redirects to the page with the product and the selected SKU |
```

Comment:
Whats is This required default values? Shouldnt be a value?

Reply:
Yes, but there are no default values for these props, so I choose to put required as a way to tell the user he must define these values. I've seen that some docs (like the ones of [react-virtualized](https://github.com/bvaughn/react-virtualized/blob/master/docs/AutoSizer.md)) that adds a column indicating if a prop is required. I'll ask in the channel what todo in these cases

Reply:
In this cases we are putting " - " on the field